### DISCUSSION TITLE
no control on people following our account

### DISCUSSION THREAD

**Discussion by lucajet** [snarfed/bridgy-fed#1526] [Ideas]
My mastodon account, bridged to bluesky, is followed there by some questionable bluesky accounts (spam, hate people).
They can follow me without even consenting themselves to be bridged also.

As i'm not on bluesky I can't block or remove them somehow from my bluesky followers.
Would be cool to do so by dm-ing the bridge

something like

block @spamgirl.bsky.social

(and by the way i'd like to remove all people that do not consent to the bridge, they shouldn't even be allowed to follow me, it's spreading confusion)

**Comment by Tamschi** [comment]
That's #1406.

**Comment by lucajet** [comment]
correct, i'm closing

**Comment by lucajet** [comment]
duplicate
